soa-derive-internal/src/iter.rs

```diff
@@ -295,4 +298,14 @@ pub fn derive(input: &Input) -> TokenStream {
             }
         }
     };
+
+    {
+        generated.append_all(quote! {
+
+            impl<'a> ::soa_derive::IntoSoAIter<'a, #name> for #slice_name<'a> {}
+
+        })
```

Comment:
Why can't this be part of the same `quote!` as the one above? I typically use this pattern to conditionally generate code

Reply:
It was likely left over from when the traits were optional, I'll add it to the previous `quote!` block